lib/clientv3/ippool_e2e_test.go

```diff
@@ -34,28 +34,32 @@ import (
 	"github.com/projectcalico/libcalico-go/lib/watch"
 )
 
-var _ = testutils.E2eDatastoreDescribe("IPPool tests", testutils.DatastoreAll, func(config apiconfig.CalicoAPIConfig) {
+var _ = testutils.E2eDatastoreDescribe("IPPool tests", testutils.DatastoreEtcdV3, func(config apiconfig.CalicoAPIConfig) {
```

Comment:
Hm, I guess there is now some IPAM code that gets hit by these tests?

I think we should be aiming to keep this set to `DatastoreAll` if possible. Can we pull out the bits that only work for etcd into a separate set of tests?

I guess CRUD on IPPools now lists IPAM blocks, which makes this fail for KDD? If so, we should track adding this back somewhere as part of implementing KDD IPAM so we don't lose these.